FirebaseFunctionsSwift/Tests/Unit/FunctionsAPITests.swift

```diff
@@ -0,0 +1,142 @@
+// Copyright 2021 Google LLC
```

Comment:
Nit: 2022

Reply:
This is a copy of the existing `FirebaseFunctions/Tests/SwiftUnit/FunctionsAPITests.swift`